src/App.js

```diff
@@ -1,13 +1,23 @@
-import "./App.css";
+import './App.css'
+import { Switch, Route } from 'react-router-dom'
+import NavBar from './Components/NavBar'
+import HomePage from './Components/HomePage'
+import Movies from './Components/Movies'
+import People from './Components/People'
+import Locations from './Components/Locations'
 
 function App() {
-  return (
-    <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
-    </div>
-  );
+	return (
+		<div className='app'>
+					<NavBar />
```

Comment:
I suggest fixing the indenting in this file. It will make it easier and more readable for other developers to work on your code if you're working on a team or need to make changes later.